TextTest will write diagnostics in /users/geoff/texttesttmp/texttest.lsfgeoff20Jan13:41:57/carmen/matador/BadCARMSYS based on file at /users/geoff/work/master/Testing/Automatic/texttest/log4py.conf
Using Application Matador, checkout /carm/proj/texttest/badtest_data
Compiling rules for Matador test-suite TargetApp
Submitting Matador test-suite TargetApp to LSF queues
  Submitting Matador test-suite ArtificialUser to LSF queues
    Submitting Matador test-case generate_in_subperiod to LSF queue short_i386_linux_RHEL
    Reading slave results for Matador test-case generate_in_subperiod (on errors.cas,output.cas,solution.cas)
    Matador test-case generate_in_subperiod had internal errors : differences in errors,output,solution
Internal ERROR (not built)
    View details(v), Save(s) or continue(any other key)?
